src/Reflectivity-VariableBreakpoint/VariableBreakpoint.class.st

```diff
@@ -95,41 +94,41 @@ VariableBreakpoint class >> watchVariableWrites: aVariableName inObject: anObjec
 
 { #category : #API }
 VariableBreakpoint class >> watchVariablesIn: aClass [
```

Comment:
I think it is better to have explicit name everywhere:
- watchVariables**InClass**: aClass
- watchVariablesInObjet: anObject

Reply:
Right, done.
I think it's a mistake to do it wihtin this PR, because it's not related to the object-centric breakpoint improvement.
But too late now, there will be a bit of noise when reviewing the PR ;)